ai_edge_torch/convert/conversion_utils.py

```diff
@@ -323,8 +319,24 @@ def convert_stablehlo_to_tflite(
     converter._experimental_enable_composite_direct_lowering = True
 
     _set_tfl_converter_quant_flags(converter, quant_config)
+    if (
+        quant_config is not None
+        and quant_config._quantizer_mode
+        == quant_config._QuantizerMode.AI_EDGE_QUANTIZER
+    ):
+      translated_recipe = translate_recipe.translate_to_ai_edge_recipe(
+          quant_config.generative_recipe
+      )
+
     _apply_tfl_backdoor_flags(converter, _tfl_converter_flags)
```

Comment:
it seems you can remove this if block , and merge the `translated_recipe` into the next if block.

Reply:
We can. The only motivation for doing this is that we can error out any invalid recipe before doing conversion, so that the user doesn't have to wait minutes to find out they have to fix their recipe.